vetapp/app/controllers/visits_controller.rb

```diff
@@ -0,0 +1,53 @@
+class VisitsController < ApplicationController
+       before_action :authenticate_user!
+
+    def index
+        @animal=Animal.all
+        @visit=Visit.all
+        @user=User.all
+    end
```

Comment:
It would make more sense for these instance variables to be plural to reflect that the values will be a collection of objects (it will return an Active Record Relation).